src/etc/inc/filter.lib.inc

```diff
@@ -201,90 +201,87 @@ function filter_core_get_default_nat_outbound_networks()
 function filter_core_rules_system($fw, $defaults)
 {
     global $config;
+    $ipv6_allow = isset($config['system']['ipv6allow']);
 
     $dhcrelay6_interfaces = plugins_run('dhcrelay_interfaces', ['inet6']);
     $dhcrelay6_interfaces = !empty($dhcrelay6_interfaces['dhcrelay']) ? $dhcrelay6_interfaces['dhcrelay'] : [];
 
     // block All IPv6 except loopback traffic
     $fw->registerFilterRule(
         1,
-        array('interface' => 'lo0', 'ipprotocol' => 'inet6', 'disabled' => isset($config['system']['ipv6allow']),
-          'descr' => 'Pass all loopback IPv6', '#ref' => 'system_advanced_network.php#ipv6allow'),
-        $defaults['pass']
-    );
-    $fw->registerFilterRule(
-        1,
-        array('ipprotocol' => 'inet6','descr' => 'Block all IPv6', 'disabled' => isset($config['system']['ipv6allow']),
-            '#ref' => 'system_advanced_network.php#ipv6allow'),
+        ['ipprotocol' => 'inet6','descr' => 'Block all IPv6', 'disabled' => $ipv6_allow,
```

Comment:
is this inverted now? it would be nicer to rename the variable and flip the logic to make this read easily... disabled => ipv6_disable

Reply:
I just kept the naming, I'm just afraid it will be confusing either way around. If we name `isset($config['system']['ipv6allow'])` `$ipv6_disable`, that's technically correct for what the rule should do (disable), but not what the ipv6 state is (enabled).....

Reply:
flipped in https://github.com/opnsense/core/pull/8010/commits/a1138e57f4b4fb9db8d6b5745e95b2f449572b52, it feels easier on the eyes, so I'm ok with it.